types/chrome/index.d.ts

```diff
@@ -8452,6 +8457,13 @@ declare namespace chrome.tabs {
      * @param callback Optional.
      */
     export function group(options: GroupOptions, callback?: (groupId: number) => void): void
+    /**
+     * Removes one or more tabs from their respective groups. If any groups become empty, they are deleted
+     * @since Chrome 88
+     * @param tabIds The tabs to ungroup.
+     * @param callback Optional. Called after the operation is completed.
+     */
+    export function ungroup(tabIds: number | number[], callback?: () => void): void
```

Comment:
Can you add an override for this function returning a Promise, similar to [line 8341](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chrome/index.d.ts#L8341)